Source/AwsCommonRuntimeKit/http/HttpClientConnectionOptions.swift

```diff
@@ -70,4 +71,41 @@ public struct HttpClientConnectionOptions {
         self.maxConnectionIdleMs = maxConnectionIdleMs
         self.shutdownCallback = shutdownCallback
     }
+
+    typealias RawType = aws_http_connection_manager_options
+    func withCStruct<Result>(shutdownOptions: aws_shutdown_callback_options, _ body: (aws_http_connection_manager_options) -> Result
+    ) -> Result {
+        return hostName.withByteCursor { hostNameCursor in
+            return withOptionalCStructPointer(proxyOptions,
+                                              proxyEnvSettings,
+                                              socketOptions,
+                                              monitoringOptions) { proxyOptionsPointer,
+                                                                   proxyEnvSettingsPointer,
+                                                                   socketOptionsPointer,
+                                                                   monitoringOptionsPointer in
+
+                var cManagerOptions = aws_http_connection_manager_options()
+                cManagerOptions.bootstrap = clientBootstrap.rawValue
+                cManagerOptions.initial_window_size = initialWindowSize
+                cManagerOptions.socket_options = socketOptionsPointer
+                cManagerOptions.tls_connection_options = tlsOptions?.rawValue
+                cManagerOptions.http2_prior_knowledge = false
+                cManagerOptions.monitoring_options = monitoringOptionsPointer
+                cManagerOptions.host = hostNameCursor
+                cManagerOptions.port = port
+                cManagerOptions.initial_settings_array = nil
+                cManagerOptions.num_initial_settings = 0
+                cManagerOptions.max_closed_streams = 0
+                cManagerOptions.http2_conn_manual_window_management = false
```

Comment:
just delete the lines where we set fields 0/false, they're not necessary anymore

Reply:
Thanks, fixed.